.vscode/settings.json

```diff
@@ -22,7 +22,7 @@
     "docker.commands.composeUp": [
         {
             "label": "Compose Up",
-            "template": "xhost +local: && USERNAME=$(whoami) USER_UID=$(id -u) USER_GID=$(id -g) ${composeCommand} ${configurationFile} up"
+            "template": "xhost +local: && USERNAME=$(whoami) USER_UID=$(id -u) USER_GID=$(id -g) RENDER_OFFSCREEN= ${composeCommand} ${configurationFile} up"
```

Comment:
Have you tested, that the space is not ignored, and the following command is used for `RENDER_OFFSCREEN`?

Reply:
Yes, the command still works and the RENDER_OFFSCREEN is ignored.